packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -133,15 +132,14 @@ class SystemRulesTable extends React.Component {
     }
 
     changePage = (page, itemsPerPage) => {
-        this.currentRows(page, itemsPerPage).then((currentRows) => {
-            this.setState(() => (
-                {
-                    page,
-                    itemsPerPage,
-                    currentRows
-                }
-            ));
-        });
+        const currentRows = this.currentRows(page, itemsPerPage);
```

Comment:
No need to assign to `currentRows`, you can directly set this in state: `currentRows: this.currentRows(page, itemsPerPage)`